File: samples/sample.py
# [TODO] Implement login functionality
def login():
    pass


# [FIXME] Crashes when username is empty
def validate(username):
    if not username:
        raise ValueError("Username required")


# [HACK] Bypassing auth for now
def fake_auth():
    return True


# [BUG] Fails on leap years
def is_leap(year):
    return year % 4 == 0 and (year % 100 != 0 or year % 400 == 0)


# [NOTE] This is a utility function
def greet(name):
    print(f"Hello, {name}")


# [OPTIMIZE] This loop is slow for large datasets
for i in range(1000000):
    pass


# [DEPRECATED] Use new_auth() instead
def old_auth():
    return False

File: samples/sample.vb
' [TODO] Add validation for user input
Dim username As String
username = Console.ReadLine()

' [NOTE] Just printing for now
Console.WriteLine("Hello, " & username)

' [FIXME] Crashes if input is null
' [HACK] For testing only – remove in production
If username = "admin" Then
    Console.WriteLine("Access granted.")
End If

' [OPTIMIZE] Reduce string concatenation calls
' [BUG] Case sensitivity issue in username check

' [DEPRECATED] Replace with GetUserInput() method

